desk/app/activity.hoon

```diff
@@ -527,14 +639,31 @@
 ++  refresh
   |=  =source:a
   =.  cor  (refresh-summary source)
-  ?+  -.source  cor
-    %channel  (refresh-summary [%group group.source])
-    %dm-thread  (refresh-summary [%dm whom.source])
-  ::
-      %thread
-    =.  cor  (refresh-summary [%channel channel.source group.source])
-    (refresh-summary [%group group.source])
-  ==
+  =.  cor
+    ?+  -.source  cor
+      %channel  (refresh-summary [%group group.source])
```

Comment:
A little bit surprised to see this wasn't using `+get-parents:src`. Could be worth refactoring lightly (now or later) so that we only ever get the source hierarchy from that utility function.